src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/Files.kt

```diff
@@ -66,12 +68,12 @@ class Files(val zip: ZipClient = Zip()) : FileSystem
                 .map { getDocumentDetails(it, documentsRoot) }
     }
 
-    override fun save(url: String, targetAbsolutePath: String)
+    override fun save(url: URL, targetAbsolutePath: String)
```

Comment:
Wonder if this method could use a more descriptive title to indicate it's the zipped archive that's being saved. saveArchive or something?